src/index.html

```diff
@@ -7,12 +7,61 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
     <h1>Product cards</h1>
+    <main class="main">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        />
+        <div class="card__title">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p>Product code: 195434</p>
```

Comment:
It's not part of the title - let's move it out of the div (which will be redundant in this case)